### PR TITLE
[ET-964] Require Event Tickets Plus 5.1.0.1+

### PR DESCRIPTION
**Pull request by sc0ttkclark**
[ET-964]

Require Event Tickets Plus 5.1.0.1+ for compatibility purposes on certain areas in Event Tickets that have direct calls to Event Tickets Plus functionality

[ET-964]: https://moderntribe.atlassian.net/browse/ET-964

### CODE REVIEW

File: readme.txt [hunk 1]
```diff
@@ -118,6 +118,10 @@ Currently, the following add-ons are available for Event Tickets:
 
 == Changelog ==
 
+= [5.0.3.1] 2020-11-19 =
+
+* Fix - Require Event Tickets Plus 5.1+ for compatibility purposes on certain areas in Event Tickets that have direct calls to Event Tickets Plus functionality.
```
Comment (sc0ttkclark) [inline readme.txt:123]:
```suggestion
* Fix - Require Event Tickets Plus 5.1+ for compatibility purposes on certain areas in Event Tickets that have direct calls to Event Tickets Plus functionality. [ET-964]
```

**Review by borkweb — DISMISSED** [review]
:+1:

**Review by bordoni — APPROVED** [review]

File: src/Tribe/Editor/Attendee_Registration.php [hunk 2]
```diff
@@ -34,7 +34,17 @@ public function hook() {
...
 	public function filter_admin_body_class( $classes ) {
-		$ar_page_slug = tribe( 'tickets.attendee_registration' )->get_slug();
+		/**
+		 * This Try/Catch is present to deal with a problem on Autoloading from version 5.1.0 ET+ with ET 5.0.3.
+		 *
+		 * @todo Needs to be revised once proper autoloading rules are done for Common, ET and ET+.
+		 */
+		try {
+			$ar_page_slug = tribe( 'tickets.attendee_registration' )->get_slug();
+		} catch( RuntimeException $error ) {
```
Comment (tr1b0t) [inline src/Tribe/Editor/Attendee_Registration.php:44]: * :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after CATCH keyword; 0 found

<sub>hash: 27d4f15c70fe111351269df589b2d7e9</sub>

File: src/Tribe/Tickets.php [hunk 3]
```diff
@@ -3590,8 +3590,17 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
...
-			$attendee_registration = tribe( 'tickets.attendee_registration' );
+			/**
+			 * This Try/Catch is present to deal with a problem on Autoloading from version 5.1.0 ET+ with ET 5.0.3.
+			 *
+			 * @todo Needs to be revised once proper autoloading rules are done for Common, ET and ET+.
+			 */
+			try {
+				/** @var \Tribe__Tickets__Attendee_Registration__Main $attendee_registration */
+				$attendee_registration = tribe( 'tickets.attendee_registration' );
+			} catch( RuntimeException $error ) {
```
Comment (tr1b0t) [inline src/Tribe/Tickets.php:3601]: * :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after CATCH keyword; 0 found

<sub>hash: c2c900740aad6902e13fce64f763bd32</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 6 error(s)

**Pull request merged**